tests/integration/test_compute_flow.py

```diff
@@ -128,6 +129,16 @@ def run_compute_test(
     print(f"got job status after requesting stop: {status}")
     assert status, f"something not right about the compute job, got status: {status}"
 
+    if restart_and_result:
+        # TODO
```

Comment:
What's the time taken TODO this? Remember, there will be 10-100 more people reading this than your writing this. So if it just takes a few min to get this test going, I strongly recommend doing so.

Reply:
no, as I explained on Slack it will take much longer, since a change in restart will most certainly break the interface and is more complicated. Restart does not work right now, so I can't test it.

Reply:
Thanks. I just saw the slack msg. OK. Then perhaps rather than say "TODO" comment it out and say "will be deprecated soon" or something.

Reply:
fyi I created this ticket as a follow-up: #232